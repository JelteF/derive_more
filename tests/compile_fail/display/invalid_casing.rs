#[derive(derive_more::Display)]
#[display(rename_all = "Whatever")]
enum Enum {
    UnitVariant,
}

fn main() {}
