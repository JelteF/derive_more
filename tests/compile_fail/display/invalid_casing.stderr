error: unexpected casing expected one of: "lowercase", "UPPERCASE", "PascalCase", "camelCase", "snake_case", "SCREAMING_SNAKE_CASE", "kebab-case", or "SCREAMING-KEBAB-CASE"
 --> tests/compile_fail/display/invalid_casing.rs:2:24
  |
2 | #[display(rename_all = "Whatever")]
  |                        ^^^^^^^^^^
